mitodl/micromasters
# CMS: Link ProgramCourse to Course

#### What are the relevant tickets?
Fix #3107

#### What's this PR do?
Show the description of the course as it was entered in wagtail.

#### How should this be manually tested?
In cms, make sure you have a ProgramPage with some ProgramCourses, where the title and description do not correspond to the actual Course in `courses`.  Run the migrations. Notice in wagtail that your program courses for that program now correspond to actual courses in that program. Now when you change the description of a course in wagtail, it should be visible in the popover when you click on the course on the program page.
